src/iris/api.py

```diff
@@ -1737,6 +1746,15 @@ def on_post(self, req, resp):
         resp.set_header('Location', '/incidents/%s' % incident_id)
         resp.body = ujson.dumps(incident_id)
 
+        # optional incident handler to do additional tasks after the incident has been created
+        if self.custom_incident_handler_module is not None:
+            connection = db.engine.raw_connection()
+            cursor = connection.cursor(db.dict_cursor)
+            cursor.execute(single_incident_query, int(incident_id))
```

Comment:
You already have the data for the incident above, so this query shouldn't be necessary

Reply:
In this case additional sql queries will be required to fetch plan and plan step info